test/Fail/Issue5468.err

```diff
@@ -1,4 +1,9 @@
 Issue5468.agda:33,1-16
-primTransp (λ i → x i) φ
-(unbox [ x₁ ]) is not usable at the required modality .
+This clause has target type primTransp (λ i → x i) φ (unbox [ x₁ ])
+which is not usable at the required modality .
```

Comment:
Maybe for relevance there should be a different printer used here.  These modalities also have long forms, see https://github.com/agda/agda/blob/4bd0e883499ae7627de05a9e1f016ad5b001da13/src/full/Agda/Syntax/Concrete/Attribute.hs#L64-L69.
So, in this case `@irrelevant` would be preferable over `.` (which I took for a punctuation mark).

You are not to blame here, rather the past (this problem already existed).  However, it would make sense to change the standard _stand-alone_ printing of `Relevance` to the long forms `@relevant`, `@irrelevant`, `@shape-irrelevant`.  In a Pi-type or telescope, the short forms ` `, `.`, `..` should still be used.

Reply:
Thoughts on using `Verbalize` from `Agda.TypeChecking.Errors` here?

Reply:
Nevermind, it's too verbose then. I'll change the printing in the suggested way.